src/RouteInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace FastRoute;
+
+interface RouteInterface
```

Comment:
This too should be dropped as we I believe we shouldn't convert this into an extension point.

Reply:
I disagree because it is a lib and I encountered this before in the CakePHP framework. Everything should be extendible, what is the downside of being able to extend it vs *not* being able to do so?